actors/miner/src/types.rs

```diff
@@ -145,6 +148,53 @@ pub struct ExpirationExtension {
     pub sectors: UnvalidatedBitField,
     pub new_expiration: ChainEpoch,
 }
+#[allow(clippy::too_many_arguments)] // validate mut prevents implementing From
+impl Into<ValidatedExpirationExtension> for ExpirationExtension2 {
```

Comment:
Nit: I find that using from/into for things like this is too opaque at the call site. I'd prefer a simple method or function `from/as_validated_expiration_extension` or similar.

Reply:
If it's a deal breaker I will do this.  I'm trying to embrace the rust way and for whatever reason like `into`s